common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleServiceMetrics.java

```diff
@@ -128,6 +142,12 @@ private static ShuffleServiceMetricsInfo getShuffleServiceMetricsInfoForCounter(
     return new ShuffleServiceMetricsInfo(name, "Value of counter " + name);
   }
 
+  private static ShuffleServiceMetricsInfo getMetricsInfoGenericValue(
```

Comment:
Although this is a private function, to be consistent with the existing methods, `getMetricsInfoGenericValue` -> `getShuffleServiceMetricsInfoForValue` might be better.